services/settings/pkg/store/defaults/defaults.go

```diff
@@ -603,6 +672,16 @@ func generateBundleProfileRequest() *settingsmsg.Bundle {
 				},
 				Value: &languageSetting,
 			},
+			{
+				Id:          SettingUUIDProfileDisableNotifications,
+				Name:        "disable email notifications",
```

Comment:
Possible to use `disable-email-notifications` here? I'd prefer to keep blanks out of identifiers that are referenced in code. The web ui uses this name to identify the setting.